ha/setup/ha_setup.py

```diff
@@ -186,15 +185,18 @@ def process(self):
         machine_id = self.get_machine_id()
         cluster_id = Conf.get(self._index, f"server_node.{machine_id}.cluster_id")
         cluster_name = Conf.get(self._index, f"cluster.{cluster_id}.name")
-        cluster_user = Conf.get(self._index, 'cortx.software.corosync.user')
+        cluster_user = Conf.get(self._index, f"cortx.software.{const.HA_CLUSTER_SOFTWARE}.user")
+        node_type = Conf.get(self._index, f"server_node.{machine_id}.type").strip()
+        self._update_env(node_type, const.HA_CLUSTER_SOFTWARE)
```

Comment:
Why are we moving this to config command instead of the init?

Reply:
All Config should be Updated before cluster configure as they may internally loading config file.

Also as config is now running on every node so they can first update local configuration file and then add node or create cluster

Reply:
ok